HOMEWORK/assets/css/main.css

```diff
@@ -0,0 +1,638 @@
+}
+
+#intro {
+    
+    background-repeat: no-repeat;
+    /* background-position: bottom right; */
+    background-position: right;
+    background-position-y: inherit;
+    /* background-size: contain; */
+}
+
+#intro .intro .menu_items{
+    display: flex;
+    justify-content: space-between;
+    padding: 40px 0px;
+}
+#intro .intro .menu_items h1{
+    font-size:20px;
+    font-family: 'Nunito-EB';
+}
+#intro .intro .menu_list ul>li{    
+    display:inline-block;    
+}
+
+#intro .intro .menu_list ul>li>a{
+    text-decoration: none;
+    color: black;
+    padding:0px 20px;    
+}
+#intro .intro .menu_list ul>li>.button{    
+    border-radius: 8px;
+    color:white;
+    padding:15px 20px;
+}
+#intro .intro .menu_list ul>li>.button:hover{
+    color: white;;
+}
+.violet{
+    background-color: #6c63ff;
+}
+.tomato{
+    background-color: tomato;
+}
+
+#intro .intro .menu_list ul>li:first-child>a{
+    color:#6c63ff;
+}
+#intro .intro .menu_list ul>li>a:hover{
+    color: #6c63ff;
+}
+
+
+
+.intro_heading_paragraph>p>span{
+    color:tomato;
+}
+.intro_heading_header{
+    font-size: 60px;
+    font-weight: 900;
+    font-family: 'Nunito-EB';
+}
+
+.intro_heading_form .form_side>ul>li{
+    display: inline-block;
+    padding: 15px;
+    border-radius: 7px 7px 0px 0px;
+}
+.intro_heading_form .form_side>ul>li:first-child{
+    background-color: #6c63ff;
+
+}
+.intro_heading_form .form_side>ul>li:first-child>a{
+    color:white;
+}
+.intro_heading_form .form_points{
+    display:flex;
+    justify-content: space-evenly;
+    background-color: #6c63ff;
+    padding: 30px 20px;
+    border-radius: 0px 0px 7px 7px;
+}
+.intro_heading_form .form_points input{
+    padding: 15px 10px;
+    border-radius: 7px;
+    width: 100px;
+    border: none;
+}
+.intro_heading_form .form_points select{
+    padding: 14px 10px;
+    border-radius: 7px;
+    width: 100px;
+    border: none;
+    color:gray;
+}
+.intro_heading_form .form_points a{
+    padding: 13px 48px;
+    border-radius: 7px;
+    width: 100px;
+    border: none;
+    line-height: 46px;
+    color: white;
+}
+#intro_features, #candidates, #subscribe{
+    background-color: #6c63ff;
+    
+   
+}
+.features_item {
+    
+    padding: 25px 28px 25px 0px;
+    color:white;
+    
+}
+
+.features_item_icon, .digits_item_icon{
+    font-size:60px;
+}
+.features_item_heading{
+    font-size: 18px;
+    padding: 15px 0px;
+    font-family: "Nunito-EB";
+}
+.features_item_description{
+    font-size:16px;
+    line-height: 20px;
+}
+#categories{
+    padding:100px 0px;
+}
+.section_heading{
+    margin:50px auto;
+    text-align: center;
+}
+.section_heading_description{
+    color:#9b9b9b;
+    font-size:18px;
+}
+.section_heading_name{
+    font-size: 30px;
+    font-family: "Nunito-EB"
+}
+.categories_item{
+    position: relative;
+    border-bottom:1px solid #9b9b9b;
+    padding: 10px 5px;   
+    transition: all 0.4s linear;
+    box-sizing: border-box; 
+    width:90%;   
```

Comment:
?

Reply:
item div-ləin altında border olduğu üçün sətrdə olan olan divlərin arasında arakəsmə olması üçün etdim. əks halda sətrlər bitişik olur div-lər.